app/assets/javascripts/app/views/header_view.js

```diff
@@ -12,12 +12,18 @@ app.views.Header = app.views.Base.extend({
     });
   },
 
-  postRenderTemplate: function(){
-    new app.views.Notifications({ el: "#notification-dropdown" });
-    this.notificationDropdown = new app.views.NotificationDropdown({ el: "#notification-dropdown" });
-    new app.views.Search({ el: "#header-search-form" });
+  postRenderTemplate: function() {
+    if (!app.collections.Notifications.instance) {
+      app.collections.Notifications.instance = new app.collections.Notifications();
+    }
```

Comment:
```javascript
setupHeader: function() {
  if(app.currentUser.authenticated()) {
    app.notificationsCollection = new app.collections.Notifications();
    app.header = new app.views.Header();
    $("header").prepend(app.header.el);
    app.header.render();
  }
},
```
in [`app/assets/javascripts/app/app.js`](https://github.com/diaspora/diaspora/blob/fa06eba/app/assets/javascripts/app/app.js#L91-L97)?